modules/virt-creating-pvcs-to-store-default-os-images.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-default-os-//images.adoc
+
+[id="virt-creating-pvcs-to-store-default-os-images_{context}"]
+= Creating ProvisionedVolumeClaims to store default OS images
+
+Follow these steps to create a ProvisionedVolumeClaim (PVC) for default OS images. After creating the PVC, you use the Web Console wizard to create new virtual machines and virtual machine templates by specifying the PVC as the *Boot Source*.
+
+.Prerequisites
+
+* You must have administrator privileges to store bootable disks as default OS images.
+
+.Procedure
+
+. In the {product-title} Web Console, click *Storage > Persistent Volume Claims* in the sidebar menu. The *Persistent Volume Claim* page displays.
+
+. Click the *Create Persistent Volume Claim* button.
+** Select the *With Form* option to create the PVC using the wizard.
+** Select the *With Data upload form* option to create the PVC with a prefilled form.
+
+. Complete the *Create Persistent Volume Claim* form to create a PVC.
```

Comment:
The most important part to mention here is to use the "browse" to find your image you want to upload and than to pick the "Attach this data to a Virtual Machine operating system" and pick the OS you want to attach this image to

Reply:
Fixed in revised version.